app/serializers/user_serializer.rb

```diff
@@ -30,6 +30,21 @@ def profile
     }
   end
 
+  def vet360_contact_information
+    person = object.vet360_contact_info
+
+    {
+      email: person.email&.details,
+      residential_address: person.address&.details,
+      mailing_address: person.mailing_address&.details,
+      mobile_phone: person.mobile_phone&.details,
+      home_phone: person.home_phone&.details,
+      work_phone: person.work_phone&.details,
+      temporary_phone: person.temporary_phone&.details,
+      fax_number: person.fax&.details
+    }
+  end
```

Comment:
@rlbaker @emersonthis @bshyong @ncksllvn - anything else you'd like to see returned in this `user` serializer profile for contact information (at least as of now)?